doc/usage.rst

```diff
@@ -109,8 +109,9 @@ The ``--show-progress`` option allows you to choose the way process progress is
 
 * ``none``: disables progress output;
 * ``dots``: multiline progress output with number of files and percentage on each line. Note that with this option, the files list is evaluated before processing to get the total number of files and then kept in memory to avoid using the file iterator twice. This has an impact on memory usage so using this option is not recommended on very large projects;
+* ``bar``: single line progress output with number of files and calculated percentage. Similar to ``dots`` output, it has to evaluate files list twice;
 
-If the option is not provided, it defaults to ``dots`` unless a config file that disables output is used, in which case it defaults to ``none``. This option has no effect if the verbosity of the command is less than ``verbose``.
+If the option is not provided, it defaults to ``bar`` unless a config file that disables output is used, in which case it defaults to ``none``. This option has no effect if the verbosity of the command is less than ``verbose``.
```

Comment:
in past, we considered changing default progress indicator as BC breaker.
not sure if we shall treat it this way, did you had a chance to think about it ?

Reply:
I would consider this as a BC break only if previous output was removed, but it's still there and can be used by passing the proper CLI option to the command. I don't consider changing default progress output (from null to progress bar) as a BC break 🙂.

Reply:
Also, progress bar [goes to stderr](https://github.com/symfony/console/blob/cd9864b47c367450e14ab32f78fdbf98c44c26b6/Helper/ProgressBar.php#L73), so it's not considered as a contract, having what you said in mind 🙂.